data/model/proxy_cache.py

```diff
@@ -0,0 +1,35 @@
+from data.database import ProxyCacheConfig
+from data.model.organization import get_organization
+
+
+def create_proxy_cache_config(
+    org_name,
+    upstream_registry,
+    upstream_registry_username=None,
+    upstream_registry_password=None,
+    staleness_period_s=0,
+    insecure=False,
+):
+    """
+    Creates proxy cache configuration for the given organization name
+    """
+    org = get_organization(org_name)
+
+    new_entry = ProxyCacheConfig.create(
+        user_id=org.id,
+        upstream_registry=upstream_registry,
+        upstream_registry_username=upstream_registry_username,
+        upstream_registry_password=upstream_registry_password,
+        staleness_period_s=staleness_period_s,
+        insecure=insecure,
+    )
+
+    return new_entry
+
+
+def get_proxy_cache_config_for_org(org_name):
+    """
+    Return the Proxy-Cache-Config associated with the given organization name, or None if it doesn't exist.
+    """
+    org = get_organization(org_name)
+    return ProxyCacheConfig.get(ProxyCacheConfig.user_id == org.id)
```

Comment:
Need to check if the given name is an org first. Unless the caller has already done so.

Reply:
Also, you need to wrap the `get()` in a try-except block, and catch `ProxyCacheConfig.DoesNotExist`, and then return None. Otherwise, an exception will be raised by Peewee

Reply:
> Need to check if the given name is an org first.

`get_organization` will raise an `InvalidOrganizationException` if `org_name` belongs to a user instead of an org.

> Also, you need to wrap the get() in a try-except block, and catch ProxyCacheConfig.DoesNotExist, and then return None. Otherwise, an exception will be raised by Peewee

Why would I _need_ to do that?

One thing I notice is that this function's doc string did not match it's behaviour - maybe that's why you're suggesting returning `None`? I've updated it to match the behaviour.